enterprise/server/remote_execution/containers/podman/podman.go

```diff
@@ -190,6 +196,16 @@ func runSociStore(ctx context.Context) {
 }
 
 func NewProvider(env environment.Env, buildRoot string) (*Provider, error) {
+	v, err := getPodmanVersion(env.GetServerContext())
+	if err != nil {
+		return nil, status.WrapError(err, "podman version")
+	}
+	podmanVersion = v
```

Comment:
Any reason not to just assign this above on line 199 and get rid of `v`?

Reply:
then I would need a `var err error` - lately I'm preferring this explicit assignment pattern to that approach, as it feels a bit too subtle that either a global or local is being assigned based on whether it's `:=` or `=`

Reply:
Ah shoot, didn't notice that `podmanVersion` isn't defined here.